calibration/CTDMOG/CGINS-CTDMOG-13638__20150623.csv

```diff
@@ -22,4 +22,4 @@ serial,name,value,notes
 37-13638,CC_ptempa0,137.4424,
 37-13638,CC_ptempa1,-0.06115682,
 37-13638,CC_ptempa2,-1.006702e-06,
-37-13638,CC_wbotc,1.5914e-07,
+37-13638,CC_wbotc,1.591357e-07,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT